docker/lb/docker-compose.yaml

```diff
@@ -175,27 +175,3 @@ services:
     depends_on:
       - avalon-lmdb
 
-  avalon-lb:
-    container_name: avalon-lb
-    image: avalon-lb-dev
-    build:
-      context: .
-      dockerfile: ./Dockerfile
-      args:
-        - http_proxy
-        - https_proxy
-        - no_proxy
-    environment:
-      - http_proxy
-      - https_proxy
-      - no_proxy
-    command: |
-      bash -c "
-        service nginx start
-        tail -f /dev/null
-        "
-    stop_signal: SIGKILL
-    depends_on:
```

Comment:
We need only depends_on with "avalon-listener-1" for difefret services in this file. Rest we can directly use from top-level docker-compose file ?

Reply:
I want to have 3 listener services running. So, do you mean to say that avalon-listener service will come from top level compose file and this file shall have only more listener services like avalon-listener-1/2/3?

Reply:
Yes. That's one possibility to reuse the top level docker-compose.yaml file. use avalon-listerner from top-level compose file and define avaon-listener-1 and avaon-listener-2 in docker-compose-lb.yaml

Reply:
Done